CHANGELOG.md

```diff
@@ -0,0 +1,115 @@
+- Rename some AAC and H264 test suites to be more precise.
+- Replace pylint and black with ruff linter and code formatter.
+- Update README.md
+- Update REPORT.md
+
+### Fixed
+- Fix bug #218: Argon AV1 test resource path don't match the downloaded resources path.
+- Check out and fix av1 argon vector path issues.
+- Issue in JSON generation scripts (Some test suites are badly generated).
+
+## [0.1.0] - 2022-12-20
+### Added
+- Refactor list as a subcommand.
+- Consolidate test suites and decoder arg into list.
+- Add copyright to every source file.
+- Add decoder argument.
+- Add running given test suites and decoders.
+- Add script use autopep8 and pylint.
+- Add a command to download test suites.
+- Add a new script that generates test suites from JCT-VT.
+- Add the JCT-VC-HEVC-V1 test suite.
+- Use md5 checksums instead of sha256.
+- Store raw decoder outputs from tests.
+- Add the JCT-VT decoder implementation.
+- gen_jct_vc: generate the H264 conformance test suite.
+- Add the H.264 reference decoder implementation.
+- Add the H264 decoder conformance test suite.
+- decoders: add Fluendo's H264 software decoders.
+- Run smoke tests with the dummy decoder.
+- Add run check for decoders.
+- Register vaapih265dec H265 decoder.
+- Remove community VA-API decoder for GStreamer 0.10.
+- Add AVC/HEVC decoder from the new VA plugin.
+- Add avdec_h264 decoder.
+- Add fluvah265dec decoder.
+- Add hev1 and hvc1 decoder.
+- Add FFmpeg vdpau decoder.
+- Add fluvah264dec decoder.
+- Add V4L2 stateless decoder support.
+- Add VP8-TEST-VECTORS.
+- Add libvpx-VP8 reference decoder.
+- Add VP9 reference decoder and test vectors.
+- Add VP9 FFmpeg decoder.
+- Add v4l2slvp8dec decoder.
+- gstreamer: Add vavp8dec support.
+- ffmpeg: Add VAAPI VP9 decoder.
+- gstreamer: Add VP9 decoders.
+- gstreamer: Add D3D11 and Libav VP8/VP9 decoders.
+- Upgrade to Python 3.7 after adding annotations.
+- Add AAC reference decoder.
+- Add support in Fluster for Windows.
+- CI: Run CI in linux and Windows.
+- Add minimal tests for GStreamer, FFmpeg and libvpx.
+- gstreamer: Add Nvidia hevc VDPAU decoder.
+- Add GStreamer NVDEC decoders to fluster.
+- Add H.266 (VTT) test suite and VVdeC reference decoder.
+- Add AV1 reference decoder and test vectors.
+- gstreamer: Add H264, H265 and VP8 V4L2 stateful decoder.
+- Add support for V4L2Codecs AV1 decoder.
+- Add support for vaapi based AV1 decoders.
+- Add AV1 gstreamer-vaapi decoder as well.
+- gstreamer: Add D3D11 AV1 decoder.
+- Add support for generating JUnit XML summary.
+- av1: add chromium 8bit and 10bit test vectors.
+- Update AV1 10bits test suites.
+- Add support for generating CSV summaries.
+- Add FFMPEG V4L2 stateful M2M decoder for VP8, VP9 and H264.
+- Add JVT Fidelity Range Extensions tests.
+- Add pyproject.toml.
+- Add 422 10bit support.
+- Add VP9 libvpx high bit depth test vectors.
+- Add VP9 libvpx 422 and 444 leftovers.
+- Create README.md.
+- Create REPORT.md.
+
+### Fixed
```

Comment:
Since 0.1.0 was the first release i would remove the section "Fixed". Whatever fixes we did were prior to the release, the user wouldn't know or be interested.